templates/default.nix

```diff
@@ -36,4 +36,6 @@ callPackage ./common.nix {
   # By default, it will just put the new depot at the end of JULIA_DEPOT_PATH.
   # You can add additional flags here.
   makeWrapperArgs = "";
+  # Extra Pkgs to precompile environment
```

Comment:
Could you add a newline before this? And remove the space between the brackets to be consistent with the rest please :)

Reply:
thanks, please review again